module/documents/actor/actor.mjs

```diff
@@ -931,6 +935,9 @@ export default class Actor5e extends Actor {
     // Add provided extra roll parts now because they will get clobbered by mergeObject below
     if ( options.parts?.length > 0 ) parts.push(...options.parts);
 
+    // Add Check Minimum
+    const abilityCheckMinimum = abl?.minimum?.check || false;
```

Comment:
Probably ok to leave these as `undefined` rather than setting them to `false`. `getMinimumAttackRoll` returns `null`, for example, and that value is passed to `d20Roll` without issue, so using any falsy value is probably fine.